tests/cases/compiler/reExportUndefined2.ts

```diff
@@ -0,0 +1,10 @@
+// @module: commonjs
+
+// @filename: a.ts
+var undefined;

```

Comment:
did you mean to give the var the type `number`? It looks like you might be testing compatibility below when you say `use(undefined)`.